app/Livewire/Project/Database/BackupEdit.php

```
@@ -98,58 +152,171 @@ public function syncData(bool $toModel = false)
 
-            $this->backup->databases_to_backup = $this->databasesToBackup;
-            $this->backup->dump_all = $this->dumpAll;
-            $this->backup->timeout = $this->timeout;
-            $this->customValidate();
-            $this->backup->save();
+                $this->backup->databases_to_backup = $this->databasesToBackup;
+                $this->backup->dump_all = $this->dumpAll;
+
+                $this->customValidate();
+                $this->backup->save();
+            }
```

PHPMD (2.15.0)

The method syncData uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)